test/test_visualization.jl

```diff
@@ -72,6 +72,59 @@ isdir(outdir) && rm(outdir, recursive=true)
     @test_nowarn_debug plot(getmesh(pd))
   end
 
+  # Run Trixi
+  @test_nowarn_debug trixi_include(@__MODULE__, joinpath(examples_dir(), "1d", "elixir_euler_blast_wave.jl"),
+                             tspan=(0,0.1))
+
+  @testset "PlotData1D, PlotDataSeries1D, PlotMesh1D" begin
+    # Constructor
```

Comment:
```suggestion

  @testset "PlotData1D, PlotDataSeries1D, PlotMesh1D" begin
    # Run Trixi
    @test_nowarn_debug trixi_include(@__MODULE__, joinpath(examples_dir(), "1d", "elixir_euler_blast_wave.jl"),
                                 tspan=(0,0.1))
                             
    # Constructor
```
This is more clear (and necessary if we ever move to something like SafeTestsets).

Reply:
Done